src/Components/Books/Books.css

```diff
@@ -13,7 +13,8 @@
 .bookitem {
   border: 1px rgb(200, 194, 194) solid;
   background-color: white;
-  display: flex;
+  display: grid;
+  grid-template-columns: 2fr 2fr 1fr;
   align-items: center;
   justify-content: space-between;
 }
```

Comment:
- [ ] Kindly add spacing between each book just as it is in the template design. Currently, you don't have a bottom margin on each book item. Please add that
- [ ] Also, I noticed the original design has a slight border radius on the corners of each book item. Kindly add that 👍 

These will do for the margin and radius
```mdx
  margin-bottom: 20px;
  border-radius: 5px;
```

- [ ] Kindly stick to the design as much as you can. Your design and color look nice but it would be good if you follow the project requirements and design the app just as in the original design. Please change the colors in the entire app to blue to match the original design.
- [ ] Also, please make the add book button to be aligned with the book items from the top just as it is in the original design. I marked the areas need fixing from the screenshots below: 👍 

PLEASE NOTE: You are highly required to match your design the same as the original design in the project requirements. 
I trust you can do it. You app looks nice, just with these changes, you good 👍 

**Your design**
![image](https://user-images.githubusercontent.com/50941074/185632186-15966fd2-9a95-49b6-b9cd-d719f882d28c.png)
 
**The original design**
![image](https://user-images.githubusercontent.com/50941074/185632614-58d408ab-5c70-4ae5-a6ab-0121fcb87250.png)

Reply:
[AmaduKamara](https://github.com/AmaduKamara) Hello, 
I will make your changes for the approval but also I don't think we're required to work like robots. For example, I don't understand why the coloring has to be exactly the same when it is my project for my portfolio. I agree that the general design should be similar but shouldn't the minute details of the project reflect my styles as a software developer? Colors and fonts for example. Why must they be exactly the same? What's the difference between blue and navy blue but personal preference. I don't understand this part.

Reply:
Hey!
Sorry about the way you feel okay.
Your design was okay but we're told that your design should be exactly as the one given and nothing else was specified as to colors or something else that should be left out. But yeah, it was a minute change you had to do and I'm sure it's not a big deal right. 
You're doing great by the way. Please keep it up. 
Good luck on your next milestone.

Reply:
@AmaduKamara thank you.